README.md

```diff
@@ -16,7 +12,14 @@ Add the following dependency to your `project.clj` file:
 [hikari-cp "1.2.4"]
 ```
 
-Plus, add the database driver you want to use.
+Note: hikari-cp targets java 8 by default. If you are using an older version of java, your `project.clj` should look more like:
+
+```clojure
+[hikari-cp "1.2.4" :exclusions [com.zaxxer/HikariCP]]
```

Comment:
Could you do `s/1\.2\.4/VERSION/` on this line? One line less to update when releasing new version...